### ISSUE TITLE
DeprecationWarning: open_binary is deprecated

### ISSUE THREAD

**Issue by cbueche** [netaddr/netaddr#288]
In `netaddr/eui/ieee.py` and `netaddr/eui/__init__.py`, there are calls to `open_binary()`, leading to this deprecation warning:

```
netaddr/eui/ieee.py:284: DeprecationWarning: open_binary is deprecated. Use files() instead.
    Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    load_index(OUI_INDEX, _importlib_resources.open_binary(__package__, 'oui.idx'))
```

Same issue with `load_index(IAB_INDEX, _importlib_resources.open_binary(__package__, 'iab.idx'))`

I'm using Python 3.11.

The code to reproduce the error :

```python
from netaddr import EUI
mac_entry='38:22:e2:8c:ef:92'
mac = EUI(mac_entry)
mac_org = mac.oui.registration(0).org
print(mac_org)
```

**Comment by jstasiak** [comment]
Hey, thank you for the report. This has been resolved in #272, which is merged but not released yet.

**Comment by jstasiak** [comment]
No worries, appreciate it! I'll release a new version soon.
